src/systems/sprites/sprite.vert

```diff
@@ -0,0 +1,28 @@
+#version 300 es
+
+uniform mat4 modelViewMatrix;
+uniform mat4 projectionMatrix;
+
+in vec4 mvCol0;
+in vec4 mvCol1;
+in vec4 mvCol2;
+in vec4 mvCol3;
+in vec3 a_vertices;
```

Comment:
I would make these singular, and I would call this one `pos`, as `vertex` is a bit ambiguous